src/psycop_feature_generation/text_models/fit_text_models.py

```diff
@@ -0,0 +1,44 @@
+"""Script for fitting text models"""
+
+from typing import Literal, Union
+
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer
+
+
+def fit_text_model(
+    model: Literal["bow", "tfidf"],
+    corpus: pd.Series,
+    ngram_range: tuple = (1, 2),
+    max_df: float = 0.95,
+    min_df: int = 2,
+    max_features: int = 500,
+) -> Union[CountVectorizer, TfidfVectorizer]:
+    """Fits a bag-of words model on a corpus
+
+    Args:
+        model (Literal[str]): Which model to use. Takes either "bow" or "tfidf".
+        corpus (Sequence[str]): The corpus to fit on
+        ngram_range (tuple, optional): The lower and upper boundary of the range of n-values for different word n-grams or char n-grams to be extracted. All values of n such such that min_n <= n <= max_n will be used. For example an ngram_range of (1, 1) means only unigrams, (1, 2) means unigrams and bigrams. Defaults to (1, 1).
+        max_df (float, optional): The proportion of documents the words should appear in to be included. Defaults to 0.95.
+        min_df (int, optional): Remove words occuring in less than min_df documents. Defaults to 2.
+        max_features (int | None, optional): If not None, build a vocabulary that only consider the top max_features ordered by term frequency across the corpus. Otherwise, all features are used. Defaults to 500.
+
+    Returns:
+        Union[CountVectorizer, TfidfVectorizer]: Fitted model
+    """
+
+    vec_type = {"bow": CountVectorizer, "tfidf": TfidfVectorizer}
+
+    # Define vectorizer
+    vec = vec_type[model](
+        ngram_range=ngram_range,
+        max_df=max_df,
+        min_df=min_df,
+        max_features=max_features,
+    )
+
+    # Fit to corpus
+    vec.fit_transform(corpus)
```

Comment:
```suggestion
    vec.fit(corpus)
```
`fit_transform` both fits the model AND applies the fitted model to the corpus (so it's essentially the same as running: 

```py
model.fit(corpus)
X = model.transform(corpus)
```